tap_stripe/__init__.py

```diff
@@ -911,8 +920,9 @@ def sync_event_updates(stream_name, is_sub_stream):
                     or when called through only child stream i.e. when parent is not selected.
     '''
     LOGGER.info("Started syncing event based updates")
-
-    date_window_size = 60 * 60 * 24 # Seconds in a day
+    event_window_size = Context.event_window_size
+    events_date_window_size = int(60 * 60 * 24 * event_window_size) # event_window_size in seconds
```

Comment:
```suggestion
    events_date_window_size = int(60 * 60 * 24 * event_window_size) # event_update_window_size in seconds
```

Reply:
Updated variable name to `events_update_date_window`